site/src/sphinx/server-annotated-service.rst

```diff
@@ -464,8 +465,36 @@ which are annotated with :api:`@RequestObject`.
 Armeria also provides built-in request converters such as, a request converter for a Java Bean,
 :api:`JacksonRequestConverterFunction` for a JSON document, :api:`StringRequestConverterFunction`
 for a string and :api:`ByteArrayRequestConverterFunction` for binary data. They will be applied
-after your request converters by default, so you can use these built-in converters by just putting
-:api:`@RequestObject` annotation on the parameters which you want to convert.
+after your request converters by default, so you can use these built-in converters by just specifying
+a parameter with desired type to be converted in the parameter list of your service method.
+
+.. code-block:: java
+
+    public class MyAnnotatedService {
+
+        // JacksonRequestConverterFunction will work for the content type of 'application/json' or
+        // one of '+json' types.
+        @Post("/hello1")
+        public HttpResponse hello1(JsonNode body) { ... }
+
+        @Post("/hello2")
+        public HttpResponse hello2(MyJsonRequest body) { ...}
+
+        // StringRequestConverterFunction will work for the content type of any of 'text'.
+        @Post("/hello3")
+        public HttpResponse hello3(String body) { ... }
+
+        @Post("/hello4")
+        public HttpResponse hello4(CharSequence body) { ... }
+
+        // ByteArrayRequestConverterFunction will work for the content type of 'application/octet-stream',
+        // 'application/binary' or none.
+        @Post("/hello5")
+        public HttpResponse hello5(byte[] body) { ... }
+
+        @Post("/hello6")
+        public HttpResponse hello6(HttpData body) { ... }
+    }
 
 In some cases, :api:`@RequestObject` annotation may have a request converter as its value.
 Assume that you have a Java class named ``MyRequest`` that it is usually able to be converted by
```

Comment:
This section could be rewritten or removed entirely.

Reply:
Will fix soon!

Reply:
Fixed